stepup/start-dev-env.sh

```diff
@@ -10,17 +10,46 @@ fi
 
 # Read the apps and their code paths from the arguments passed to the script
 docker_compose_args=()
+
+# Keep a counter of the number of dev-envs to override
+number_of_dev_envs=0
 for arg in "$@"; do
 	app=$(echo $arg | cut -d ':' -f 1)
 	path=$(echo $arg | cut -d ':' -f 2)
 	echo "export ${app^^}_CODE_PATH=${path}" >>.start-dev-env-vars
+	# Keep a listing of all apps that are started in dev mode, for feedback purposes
+	echo "${app^^}: ${path}" >>.start-dev-env-listing
 	docker_compose_args+=("-f ./${app}/docker-compose.override.yml")
+	let number_of_dev_envs=number_of_dev_envs+1
 done
 
+# Because numbering is off by one, reference the next arg
+let number_of_dev_envs=number_of_dev_envs+1
 # Read the generated env file with the apps and their code paths
```

Comment:
I thing you can simplefy the argument processing using `shift` (https://tldp.org/LDP/Bash-Beginners-Guide/html/sect_09_07.html)
See e.g. https://github.com/OpenConext/Stepup-Build/blob/master/stepup-build.sh

Reply:
@pmeulen I see how this could work, I'm going to attempt to utilize that method. I'll be sure to check in with you if I get stranded.